test/simulation_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'minitest/autorun'
+require 'minitest/unit'
+require 'mocha/minitest'
+require './lib/cell'
+require './lib/board'
+require './lib/simulation'
+
+class SimulationTest < MiniTest::Test
+
+  def setup
+    @simulation = Simulation.new
+  end
+
+  def test_instantiation
+    assert_instance_of Simulation, @simulation
+    assert_instance_of Board, @simulation.board
+    assert_instance_of Cell, @simulation.board.cells.first.first
+    
+    # center_coordinate will be created as [1][1] by default due to 3x3 grid
+    center_coordinates = { row : 1, column: 1 }
+    assert_equal center_coordinates, @simulation.target_cell
```

Comment:
Wanting to pass hard coded indices for a simulation's target cell given the prompt originally asks for a simulation to focus on a center cell